doc/release.md

```diff
@@ -7,15 +7,33 @@ So, you want to release the `X.Y.Z` version of astroid ?
 (Consider triggering the "release tests" workflow in GitHub Actions first.)
 
 1. Check if the dependencies of the package are correct
-2. Check the result (Do `git diff vX.Y.Z-1 ChangeLog` in particular).
-3. Install the release dependencies `pip3 install pre-commit tbump`
-4. Bump the version and release by using `tbump X.Y.Z --no-push`.
-5. Push the tag.
-6. Release the version on GitHub with the same name as the tag and copy and paste the
+2. (If you're releasing a minor (`X.Y.0`), remove the empty changelog for the last
+   unreleased patch version `X.Y-1.Z'`.)
+3. Check the result of `git diff vX.Y.Z-1 ChangeLog` or `git diff vX.Y-1.Z' ChangeLog`.
+4. Install the release dependencies `pip3 install pre-commit tbump`
+5. Bump the version and release by using `tbump X.Y.Z --no-push`.
+6. Check the result and push the tag.
+7. Release the version on GitHub with the same name as the tag and copy and paste the
    appropriate changelog in the description. This trigger the pypi release.
+8. (If you just released a minor (`X.Y.0`), also release what was not yet released on
```

Comment:
This should probably be done before the minor is released.
So: `2.9.0` > `2.9.1` > `2.10.0`.

Reply:
[90d3373](https://github.com/PyCQA/astroid/pull/1438/commits/90d3373a1a6a8692896a494c8a28f5811b02e433)